RELEASE_PROCESS.md

```diff
@@ -39,9 +39,19 @@ dnf makecache
 dnf upgrade tektoncd-cli
 ```
 
-- Make an update for [homebrew-core](https://github.com/Homebrew/homebrew-core/blob/master/Formula/tektoncd-cli.rb) for tektoncd-cli formula, and make sure you get the proper `sha256sum` and update it like this:
-
-  https://github.com/Homebrew/homebrew-core/pull/46492
+- Make an update for [homebrew-core](https://github.com/Homebrew/homebrew-core/blob/master/Formula/tektoncd-cli.rb) for tektoncd-cli formula.
+
+  * Make sure you get the proper `sha256sum` You need to download the `Source Code (tar.gz)` from the release and hit below command for the respective release
+  
+    ```shell script
+    openssl dgst --sha256 cli-0.7.1.tar.gz 
```

Comment:
I guess openssl works but so should be the `sha256sum`  tool

Reply:
yaa both of them works

Reply:
Maybe more generally to get sha256 and say the example below uses `openssl`? I'm not sure which too is more commonly used.

Reply:
i have changed the command to `sha256sum`